.github/workflows/build-and-push-assets.yml

```
@@ -53,6 +53,16 @@ on:
         type: string
         default: ''
         required: false
+      PHP_VERSION:
+        description: PHP version with which the WP CLI is going to be executed.
```

Perhaps it should be something more generic.
```suggestion
        description: PHP version with which PHP tools are going to be executed.
```